planner/core/planbuilder.go

```diff
@@ -4714,6 +4714,9 @@ func buildShowSchema(s *ast.ShowStmt, isView bool, isSequence bool) (schema *exp
 	case ast.ShowBindings:
 		names = []string{"Original_sql", "Bind_sql", "Default_db", "Status", "Create_time", "Update_time", "Charset", "Collation", "Source"}
 		ftypes = []byte{mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeDatetime, mysql.TypeDatetime, mysql.TypeVarchar, mysql.TypeVarchar, mysql.TypeVarchar}
+	case ast.ShowBindingCacheStatus:
+		names = []string{"num_bindings", "total_bindings", "memory_usage", "memory_capacity"}
```

Comment:
I think the difference between the first two columns is not so straightforward. How about name them something like "bindings_in_cache" and "bindings_in_table"?
And I think "memory_capacity" can be named "memory_quota" to be consistent with the config variable.